cli/src/pcluster/models/cluster_slurm.py

```diff
@@ -123,3 +123,28 @@ def __init__(
                         os=self.image.os,
                         architecture=self.architecture,
                     )
+
+    @property
+    def compute_subnet_ids(self):
+        """Return the list of all compute subnet ids in the cluster."""
+        return list(
+            {
+                subnet_id
```

Comment:
If we wrap the element in list, we could do `subnet_id.value ...for subnet_id in queue.networking.subnet_ids`

Reply:
See my comment above and of course let's follow up if you think I am missing something.